packages/core/src/base/typography/typography.example.html

```diff
@@ -0,0 +1,11 @@
+<h1 class="ds-display">Display</h1>
+<h1 class="ds-title">Title</h1>
+<h1 class="ds-h1">Heading 1</h1>
+<h1 class="ds-h2">Heading 2</h1>
+<h1 class="ds-h3">Heading 3</h1>
+<h1 class="ds-h4">Heading 4</h1>
+<h1 class="ds-h5">Heading 5</h1>
+<h1 class="ds-h6">Heading 6</h1>
```

Comment:
separated the example and remove some of the extra redundant paragraphs in the example